### PR TITLE
Add sqlite functions

### PR DESCRIPTION
**Pull request by winkbrace**
This PR adds 3 common functions for SQLite: IFNULL, REPLACE and ROUND

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /composer.lock
 /tests/phpunit.xml
 /vendor
+.idea
```
Comment (stevelacey) [inline .gitignore]: You should add this into your global gitignore, not here: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Comment by stevelacey** [comment]
Looks good, can you update the `README.md` and `config/sqllite.yml` to describe your new functions – thanks

File: .gitignore (hunk 1 above)
Comment (winkbrace) [inline .gitignore]: Ok, thanks for the link. Will do.

**Comment by winkbrace** [comment]
Travis is failing, because one of the tests verifies the current date, but there is some time between the generating of the result and the assertion.

This is unrelated to my PR

**Comment by stevelacey** [comment]
Yes, it's very annoying, I've given it a kick

**Comment by stevelacey** [comment]
@winkbrace your other PR conflicted this one once merged, resolve and this can go in too :smile: thanks!

**Comment by winkbrace** [comment]
Updated :)
Thanks for accepting.

**Comment by stevelacey** [comment]
Thanks for your contribution!

**Comment by stevelacey** [comment]
Tagged as `v1.0.6`

**Pull request merged**